apollo-federation/src/supergraph/mod.rs

```diff
@@ -1666,16 +1636,6 @@ fn get_subgraph<'subgraph>(
     })
 }
 
-fn get_index_from_subgraph_name<'a>(
```

Comment:
Refactors:
- To get the enum value more easily for a subgraph name, it's been added to `FederationSubgraph`.